proto/gcpb.proto

```diff
@@ -12,15 +12,15 @@ option (rustproto.lite_runtime_all) = true;
 option java_package = "org.tikv.kvproto";
 
 service GC {
-  rpc GetAllServiceGroups(GetAllServiceGroupsRequest) returns (GetAllServiceGroupsResponse) {}
+  rpc GetAllKeySpaces(GetAllKeySpacesRequest) returns (GetAllKeySpacesResponse) {}
 
-  rpc GetMinServiceSafePointByServiceGroup(GetMinServiceSafePointByServiceGroupRequest) returns (GetMinServiceSafePointByServiceGroupResponse) {}
+  rpc GetMinServiceSafePointByKeySpace(GetMinServiceSafePointByKeySpaceRequest) returns (GetMinServiceSafePointByKeySpaceResponse) {}
 
-  rpc UpdateGCSafePointByServiceGroup(UpdateGCSafePointByServiceGroupRequest) returns (UpdateGCSafePointByServiceGroupResponse) {}
+  rpc UpdateGCSafePointByKeySpace(UpdateGCSafePointByKeySpaceRequest) returns (UpdateGCSafePointByKeySpaceResponse) {}
 
-  rpc UpdateServiceSafePointByServiceGroup(UpdateServiceSafePointByServiceGroupRequest) returns (UpdateServiceSafePointByServiceGroupResponse) {}
+  rpc UpdateServiceSafePointByKeySpace(UpdateServiceSafePointByKeySpaceRequest) returns (UpdateServiceSafePointByKeySpaceResponse) {}
```

Comment:
I think we do not need to stressed that the service's safepoint with `KeySpace` because the definiation of `service` is  in the namespace of `key-space`
```suggestion
  rpc UpdateServiceSafePoint(UpdateServiceSafePointByKeySpaceRequest) returns (UpdateServiceSafePointResponse) {}
```